tools/castle-editor/code/formchooseproject.pas

```diff
@@ -70,6 +73,24 @@ implementation
 
 { TChooseProjectForm ------------------------------------------------------------- }
 
+procedure TChooseProjectForm.Show;
+begin
+  {$IFDEF WINDOWS}
+  Application.ShowMainForm := True;
+  {$ELSE}
+  TForm(Self).Show;
+  {$ENDIF}
```

Comment:
Instead of calling `TForm(Self).Show` I think it's a bit cleaner to call `inherited Show`. This more clearly shows the intention -- you don't want to call your own method, you want `Show` of  ancestor.

Same note about `TForm(Self).Hide` below.

Reply:
Haven't used Pascal for 2 years and I already forgot we can use inherited to access ancestor methods... Thanks for your remarks!